convert2rhel/unit_tests/grub_test.py

```diff
@@ -159,8 +159,16 @@ def test_get_device_number(monkeypatch, caplog, expected_res, device, exc, subpr
         assert len(caplog.records) == 0
 
 
-def test_get_device_number_no_output(monkeypatch):
-    monkeypatch.setattr("convert2rhel.utils.run_subprocess", RunSubprocessMocked(return_value=("", 0)))
+@pytest.mark.parametrize(
+    ("output"),
+    (
+        (""),
+        (" "),
+        ("\n"),
```

Comment:
Not sure if it is worth it, but I would include those cases: 

```suggestion
        ("\n"),
        (" \n \r"),
        ("\r"),

```